src/Microsoft.AspNetCore.DataProtection.EntityFrameworkCore/IDataProtectionKeyContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace Microsoft.AspNetCore.DataProtection.EntityFrameworkCore
+{
+    /// <summary>
+    /// Interface used to store instances of <see cref="DataProtectionKey"/> in a <see cref="DbContext"/>
+    /// </summary>
+    public interface IDataProtectionKeyContext
+    {
+        /// <inheritdoc />
+        DbSet<DataProtectionKey> DataProtectionKeys { get; }
```

Comment:
Requiring a `DbSet<DataProtectionKey>` on the `DbContext` seems a bit rigid. Could just use `DbContext.Set<DataProtectionKey>()` instead.